releasenotes/notes/snmp_add_agent_host_as_tag-fb8f9987d9ea8522.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    [corechecks/snmp] Add agent host as tag when ``use_device_id_as_hostname`` is enabled
```

Comment:
```suggestion
    [corechecks/snmp] Add agent host as tag when `use_device_id_as_hostname` is enabled.
```

Reply:
@sarina-dd Thanks for the review. 

Actually, this is a `.rst`/reStructuredText format file and it requires double black tick to format as code.

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/inline.html